metrics/density/footprint_data.sh

```diff
@@ -316,12 +316,13 @@ function grab_system() {
 			"avail": $avail,
 			"avail_decr": $avail_decr,
 			"cached": $cached,
-			"smem_free": $smem_free,
 			"free_decr": $free_decr,
 			"anon": $anon,
 			"mapped": $mapped,
 			"meminfo_cached": $meminfo_cached,
-			"Units": "KB"
+			"Units": "KB",
```

Comment:
I think having two entries both called `Units` is not valid JSON. You'll either have to break the `B` version out into a new section, or probably better is to scale it down to KB in this section. If you move it to a new place in the JSON, you will probably also break the evaluators (any metrics or report generator that processes this data).

Reply:
@grahamwhaley changes applied